cfgov/data_research/scripts/process_mortgage_data.py

```diff
@@ -106,21 +106,23 @@ def process_source(
                     (datetime.datetime.now() - starter),
                     len(new_objects)))
     if dump_slug:
-        rows = []
-        for obj in new_objects:
-            rows.append([
-                obj.pk,
-                obj.fips,
-                "{}".format(obj.date),
-                obj.total,
-                obj.current,
-                obj.thirty,
-                obj.sixty,
-                obj.ninety,
-                obj.other,
-                county.pk
-            ])
-        dump_as_csv(rows, dump_slug)
+        dump_as_csv(
```

Comment:
This is a significant speedup to this script because instead of first constructing a huge array in memory and then writing it, we just construct as needed while we write row by row.

Reply:
very nice.